geonode/base/api/views.py

```diff
@@ -436,6 +437,9 @@ def set_thumbnail_from_bbox(self, request, resource_id):
         except ResourceBase.DoesNotExist:
             logger.error(f"Resource selected with id {resource_id} does not exists")
             return Response(data={"message": f"Resource selected with id {resource_id} does not exists"}, status=404, exception=True)
+        except ThumbnailError as e:
```

Comment:
Please @mattiagiupponi let's implement a more specific exception. This is too much generic and we risk to hide other types of errors.